### ISSUE TITLE
Problema al mostrar imágenes en un QLabel (Contenidos semana 7)

### ISSUE THREAD

**Issue by SVenegasA** [IIC2233/Syllabus#132]
### Prerrequisitos
(Marcar colocando una X entre los corchetes los ítems que ya hiciste, así: "[X]")

* [X] Leí las reglas del foro (https://github.com/IIC2233/syllabus/issues/4)
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues)
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al Jefe de Bienestar o al correo del curso (https://iic2233.github.io/contacto/)
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla
* [X] De ser necesario, colocaré código simple que permita explicar mi problema o duda

### Duda

Estoy usando el siguiente código (Visto en el ejemplo de cómo mostrar imágenes, del archivo "2-elementos-gráficos-básicos.ipynb"):
import sys
import os
from PyQt5.QtWidgets import QApplication, QWidget, QLabel, QLineEdit
from PyQt5.QtGui import QPixmap

class MiVentana(QWidget):

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.init_gui()

    def init_gui(self):
        """
        Este método inicializa la interfaz y todos sus widgets.
        """

        # Ajustamos la geometría de la ventana y su título
        self.setGeometry(200, 100, 200, 200)
        self.setWindowTitle('Ventana con imagen')

        # Creamos el QLabel que contendrá la imagen y definimos su tamaño
        self.label = QLabel(self)
        self.label.setGeometry(50, 50, 100, 100)

        # Escribimos la ruta al archivo que contiene la imagen.
        # La imagen obtenida en https://en.wikipedia.org/wiki/Python_(genus)
        ruta_imagen = os.path.join('img', 'python.jpg')

        # Cargamos la imagen como pixeles
        pixeles = QPixmap(ruta_imagen)

        # Agregamos los pixeles al elemento QLabel
        self.label.setPixmap(pixeles)

        # Finalmente, ajustamos tamaño de contenido al tamaño del elemento (100 x 100)
        self.label.setScaledContents(True)

        # Una vez que fueron agregados
        # todos los elementos a la ventana la
        # desplegamos en pantalla
        self.show()

if __name__ == '__main__':
    app = QApplication([])
    ventana = MiVentana()
    sys.exit(app.exec_())

**Ya me aseguré de que el archivo python.jpg estuviera en la misma carpeta donde se encuentra el script y de que el archivo tuviera el mismo nombre** sin embargo no me muestra ninguna imágen en el QLabel. **¿Dónde podría estar el error?**

**Comment by nicoabarca** [comment]
Hola @SVenegasA , ojo :eye: que la ruta de la imagen en el código es `ruta_imagen = os.path.join('img', 'python.jpg')`, esto quiere decir que la imagen `python.jpg` esta dentro de una carpeta llamada `img`, tu directorio debería estar estructurado de la siguiente manera.
```
.
| script.py
| img/python.jpg
```
Espero eso resuelva tu problema 😄

**Comment by SVenegasA** [comment]
Sí, el problema era que no entendía los párametros que recibía os.path.join(). Ahora que le quité el "img" a mí codigo sí se muestra la imagen. Gracias

**Comment by tzehnder** [comment]
Hola tengo una duda similar, utilizo exactamente el mismo codigo a diferencia de que en mi escritorio si utilizo la carpeta img, tambien probe por las dudas dejando el archivo python.jpg en la misma carpeta que el codigo y eliminando del join 'img'  pero tampoco me mostro la imagen, tambien me ha pasado con la mayoria de los otros ejemplos o cuando queria agregar un logo a los inicios de sesion, sabes cual podria ser el error?
